pantab/_hyper_util.py

```diff
@@ -0,0 +1,28 @@
+from contextlib import contextmanager
+from typing import Optional
+
+import tableauhyperapi as tab_api
+
+
+# As soon as we drop support for Python 3.6: Use `contextlib.nullcontext` instead
```

Comment:
Thanks for the note. The next release I make I actually plan to drop 3.6 as a lot of the scientific stack only supports 3 python versions at a time, which today is 3.7, 3.8 and 3.9

OK to leave for now though - just providing as FYI